### ISSUE TITLE
python create_submission.py EPIC_Sounds_recognition_test_timestamps.pkl test_results.zip

### ISSUE THREAD

**Issue by haoshuai714** [epic-kitchens/C9-epic-sounds#1]
if i run this command:python create_submission.py EPIC_Sounds_recognition_test_timestamps.pkl test_results.zip
it will have a test_result.zip?

But, i have a problem:
usage: create_submission.py [-h] [--sls-pt {0,1,2,3,4,5}] [--sls-tl {0,1,2,3,4,5}] [--sls-td {0,1,2,3,4,5}] [--t_mod {0,1,2}] [--force] scores scores_json_zip version

**Comment by JacobChalk** [comment]
Valid submissions require additional flags describing the submitted models training details. These relate to the three supervision level scales and an additional modality indicator. Further information is provided in the "help" section of the args, within the code, or alternatively you can read about what each scale and respective levels mean [here](https://arxiv.org/abs/2008.09890). `t_mod` is a flag specific to this challenge, indicating if the model was trained on:
0 - Audio-only data, 1 - Video-only data, 2 - Audio-Visual data.

TLDR; the flags --sls-pt, --sls-tl, --sls-td and --t_mod are required inputs to generate the submission file. Re-run with `python create_submission.py EPIC_Sounds_recognition_test_timestamps.pkl test_results.zip --sls-pt <SCALE> --sls-tl <SCALE> --sls-td <SCALE> --t_mod <DATA_TYPE>`

**Comment by haoshuai714** [comment]
Running the script is wrong again, create_submission.py: error: the following arguments are required: version。

Can you provide a complete script that can generate commit files, thanks

**Comment by haoshuai714** [comment]
i think additional flags as follow:
"version": "0.1",
  "challenge": "audio_based_interaction_recognition",
  "sls_pt": -1,
  "sls_tl": -1,
  "sls_td": -1,
  "t_mod": -1
Right?

**Comment by JacobChalk** [comment]
The version argument is defaulted and won't need to be included. The sls flags need to be in the range [0, 5] and t_mod needs to be in the range [0, 2]. `create_submission.py -h` should explain each arg and what each scale means.

**Comment by haoshuai714** [comment]
thanks！
